kubridge/VITABUILD

```diff
@@ -1,8 +1,8 @@
 pkgname=kubridge
 pkgver=9999
 pkgrel=1
-url="https://github.com/TheOfficialFloW/kubridge"
-source=("git+https://github.com/TheOfficialFloW/kubridge.git")
+url="https://github.com/bythos14/kubridge"
+source=("git+https://github.com/bythos14/kubridge.git")
```

Comment:
Can you use a specific commit? You can add `#commit=` and then the commit sha to the end of the url.

Reply:
Why so? I kinda thought not pinning git repos is a feature, so that we get the most fresh packages on every build?

Reply:
9999 packages shouldn't be pinned to commits. Their whole point is to always point to latest master.

Using 9999 is discouraged, however, but with lack of active maintainership to merge updates and rapid changes in upstream (like vitagl, for example) - it's ok, imho.

Reply:
I'm not a fan of this policy, but I do understand how it came to be. I think a date based format based on the git log using the version function would be better. Here is an example of a package that uses the commit number and sha: https://github.com/pspdev/psp-packages/blob/1eb9bb2f05f7076458660025907e011e473e002d/libpspvram/PSPBUILD#L15

A package manager could not work with 9999 and the same version number of a package being different can be confusing. I would consider changing the policy. If that's not wanted, I'll close these comments, though.

Reply:
9999 is a standard arch PKGBUILD (which vitabuilds are based on) concept for "rolling"  packages.

So, if @sleirsgoevy plans to update uvdb frequently, 9999 with master is fine.
If not - proper versioning with tag/commit pinning should be used (and NOT 9999).
Same goes for kubridge

Reply:
Arch Linux does not use this convention for any package in any of the official repositories. I think it is unique to the VitaSDK. Their guidelines on git based packages can be found [here](https://wiki.archlinux.org/title/VCS_package_guidelines).

Reply:
> plans to update uvdb frequently

Probably not. There will probably be some updates, but not that frequent to justify 9999.

> If not - proper versioning with tag/commit pinning should be used (and NOT 9999).

I see. Will update the VITABUILD accordingly.

Reply:
> Arch Linux does not use this convention for any package in any of the official repositories. I think this approach is unique to the VitaSDK. The Arch Linux guidelines on git based packages can be found [here](https://wiki.archlinux.org/title/VCS_package_guidelines).

oh, right. that's gentoo thing. but i think it's still used by some of arch packages.
(not that it really matters for vitasdk, since there's zero package management)